packages/samples/headless-react/src/components/search-box/search-box.class.tsx

```diff
@@ -0,0 +1,64 @@
+import {Component} from 'react';
+import {
+  buildSearchBox,
+  SearchBox as HeadlessSearchBox,
+  SearchBoxOptions,
+  SearchBoxState,
+  Unsubscribe,
+} from '@coveo/headless';
+import {engine} from '../../engine';
+
+export class SearchBox extends Component {
+  private controller: HeadlessSearchBox;
```

Comment:
One thing I forgot: should `controller` be `readonly`?

Reply:
Yep, it could be. I will merge this PR as-is to not restart another CI, but feel free to change it in your PR,